### PR TITLE
Add new board for VEkTORF722 PYI

### PR DESCRIPTION
**Pull request by Abhigyan-R**
Added new target for PYI manufacturer as discussed on discord.

### CODE REVIEW

File: configs/VEkTORF722/config.h [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU   STM32F7X2
+#define BOARD_NAME      VEkTORF722
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:25]:
```suggestion
#define BOARD_NAME      VEKTORF722
```

File: configs/VEkTORF722/config.h [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+#define MANUFACTURER_ID PYI
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_FLASH
+#define USE_FLASH_W25Q128FV
+#define USE_FLASH_W25N01G
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:34]: This flash chip performs badly so it's not recommended / unsupported

File: configs/VEkTORF722/config.h [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_MAX7456
+
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:38]: Excessive white space 
```suggestion
```

File: configs/VEkTORF722/config.h [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+#define GYRO_1_EXTI_PIN PC4
+#define GYRO_1_CS_PIN PA4
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PC8,  2,  0) \
+    TIMER_PIN_MAP( 1, PC9,  2,  0) \
+    TIMER_PIN_MAP( 2, PA8,  1,  0) \
+    TIMER_PIN_MAP( 3, PA9,  1,  0) \
+    TIMER_PIN_MAP( 4, PB0,  2,  0) \
+    TIMER_PIN_MAP( 5, PB1,  2,  -1) \
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:88]:
```suggestion
    TIMER_PIN_MAP( 5, PB1,  2, -1) \
```

File: configs/VEkTORF722/config.h (hunk 1 above)
Comment (nerdCopter) [inline configs/VEkTORF722/config.h:25]: capitals only

File: configs/VEkTORF722/config.h [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    TIMER_PIN_MAP( 6, PA10, 1,  0) \
+    TIMER_PIN_MAP( 7, PB4,  1,  -1) \
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:90]:
```suggestion
    TIMER_PIN_MAP( 7, PB4,  1, -1) \
```

File: configs/VEkTORF722/config.h [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    TIMER_PIN_MAP( 8, PB3,  1,  0)
+
+#define ADC3_DMA_OPT 0
+
+#define MSP_UART SERIAL_PORT_USART5
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:95]: Is this for DJI ?

File: configs/VEkTORF722/config.h [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+#define DEFAULT_CURRENT_METER_SCALE 250
+#define BEEPER_INVERTED
+#define MAX7456_SPI_INSTANCE SPI2
+#define FLASH_SPI_INSTANCE SPI2
+#define USE_SPI_GYRO
+#define GYRO_1_SPI_INSTANCE SPI1
+#define GYRO_1_ALIGN CW90_DEG
+#define PINIO1_BOX 40
+#define PINIO2_BOX 0
+#define PIONIO2_CONFIG 129
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:110]: No newline on EOF
```suggestion
#define PIONIO2_CONFIG 129

```

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/VEkTORF722/config.h [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU   STM32F7X2
+#define BOARD_NAME      VEkTORF722
+#define MANUFACTURER_ID PYI
```
Comment (haslinghuis) [inline configs/VEkTORF722/config.h:26]: Unknown manufacturer. Needs to be 4 characters and be added to manufacturers.md
```suggestion
#define MANUFACTURER_ID PYI
```

**Comment by nerdCopter** [comment]
first time i see a 3-cahracter manufacturer ID. is this valid @haslinghuis ?

**Comment by haslinghuis** [comment]
@Abhigyan-R please give a proper introduction as manufacturer before submitting new board etc.

We need
- create a manufacturer ID
- manufacturer website or product info at least.
- manufacturer engineer for maintenance (if applicable)
- development samples for verification.

File: configs/VEkTORF722/config.h (hunk 8 above)
Comment (freasy) [reply configs/VEkTORF722/config.h:26]: the manufacturer is named "PYI" - what could the 4th character be?

Comment (haslinghuis) [reply configs/VEkTORF722/config.h:26]: 26 posibilities :)

Comment (freasy) [reply configs/VEkTORF722/config.h:26]: Neither fit into the manufacturers name

File: configs/VEkTORF722/config.h (hunk 6 above)
Comment (Abhigyan-R) [reply configs/VEkTORF722/config.h:94]: Yes it will be used for DJI system

**Review by Abhigyan-R — COMMENTED** [review]
Changes made as requested

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**